src/System.Management.Automation/engine/runtime/Binding/Binders.cs

```diff
@@ -6533,6 +6534,14 @@ public override DynamicMetaObject FallbackInvoke(DynamicMetaObject target, Dynam
 
     internal sealed class PSInvokeMemberBinder : InvokeMemberBinder
     {
+        private const string WhereMagicMethodName = "Where";
+        private const string WhereMagicMethodPSName = $"PS{WhereMagicMethodName}";
+        private const string ForeachMagicMethodName = "Foreach";
+        private const string ForeachMagicMethodPSName = $"PS{ForeachMagicMethodName}";
```

Comment:
Those are only used in the declaration of `s_whereSearchValues` and `s_foreachSearchValues` below, so it doesn't seem to be needed to have them as constant variables.